discovery-infra/tests/base_test.py

```diff
@@ -345,3 +348,27 @@ def kube_api_context(self, kube_api_client):
 
         with kube_api_context:
             yield kube_api_context
+
+    @pytest.fixture(scope="function")
+    def update_olm_config(self) -> Callable:
+        def update_config(tf_config, cluster_config, operators=None):
+            if operators is None:
+                operators = operators_utils.parse_olm_operators_from_env()
+
+            tf_config.worker_memory = operators_utils.resource_param(tf_config.worker_memory, consts.OperatorResource.WORKER_MEMORY_KEY, operators)
+            tf_config.master_memory = operators_utils.resource_param(tf_config.master_memory, consts.OperatorResource.MASTER_MEMORY_KEY, operators)
+            tf_config.worker_vcpu = operators_utils.resource_param(tf_config.worker_vcpu, consts.OperatorResource.WORKER_VCPU_KEY, operators)
+            tf_config.master_vcpu = operators_utils.resource_param(tf_config.master_vcpu, consts.OperatorResource.MASTER_VCPU_KEY, operators)
+            tf_config.workers_count = operators_utils.resource_param(tf_config.workers_count, consts.OperatorResource.WORKER_COUNT_KEY, operators)
+            tf_config.worker_disk = operators_utils.resource_param(tf_config.worker_disk, consts.OperatorResource.WORKER_DISK_KEY, operators)
+            tf_config.master_disk = operators_utils.resource_param(tf_config.master_disk, consts.OperatorResource.MASTER_DISK_KEY, operators)
+            tf_config.master_disk_count = operators_utils.resource_param(tf_config.master_disk_count, consts.OperatorResource.MASTER_DISK_COUNT_KEY, operators)
+            tf_config.worker_disk_count = operators_utils.resource_param(tf_config.worker_disk_count, consts.OperatorResource.WORKER_DISK_COUNT_KEY, operators)
+
+            cluster_config.workers_count = operators_utils.resource_param(cluster_config.workers_count, consts.OperatorResource.WORKER_COUNT_KEY, operators)
+            cluster_config.nodes_count = cluster_config.masters_count + cluster_config.workers_count
+            cluster_config.olm_operators = [operators]
+
+            return tf_config, cluster_config
```

Comment:
You don't have to return the configurations. It's update function.

Reply:
It makes test code easy if I return them in this way. It makes sense to make to merge both.